src/webgpu/shader/execution/builtin/section16_01.spec.ts

```diff
@@ -0,0 +1,55 @@
+export const description = `WGSL section 16.1 execution test`;
```

Comment:
WGSL section numbers probably aren't going to stay stable. For the same reason, URLs link to sections by name, e.g.:
https://gpuweb.github.io/gpuweb/wgsl/#logical-builtin-functions
That's probably necessary here too.